src/proxy/http/HttpSM.cc

```diff
@@ -5884,9 +5884,6 @@ HttpSM::handle_post_failure()
 void
 HttpSM::handle_http_server_open()
 {
-  // The request is now not queued. This is important because server retries reuse the t_state.
-  t_state.outbound_conn_track_state.dequeue();
-
```

Comment:
This might seem like a concerning removal of a use of dequeue, but note that enqueue was never called. Thus this call was always a no-op.